<launch>
     <include file="$(find vrep_ros_teleop)/launch/teleop_mux.launch"/>

    <node pkg="shore_follower_truck" type="shore_follower_drive_truck.py" name="shore_follower_drive_truck" output="screen">
      <param name="model_dir" value="/home/GTL/lschwing/catkin_ws/src/shore_follower_truck/output/models/step_4500" />
      <!-- rotation speed corresponding to left or right commands -->
      <param name="twist_factor" value="0.5" />
      <param name="linear_vel" value="0.2" />

      <remap from="/shore_follower_drive_truck/image" to="/vrep/visionSensor"/>
      <remap from="/shore_follower_drive_truck/twist" to="/vsv_driver/twistCommand"/>
  </node>
</launch>
